image/image_test.go

```diff
@@ -678,3 +680,22 @@ func (s *imageSuite) TestInstallCloudConfigWithCloudConfig(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(content, DeepEquals, canary)
 }
+
+func (s *imageSuite) TestNewToolingStoreWithAuth(c *C) {
```

Comment:
Commenting here because it's about tests. I think we also need tests for bad filename in the env var and invalid json.